### PR TITLE
Sample vSphere Python SDK program to perform snapshot operations

### PR DESCRIPTION
**Pull request by Abdul-Anshad-A**
Sample vSphere Python SDK program to perform snapshot operations like "create/remove/revert/list_all/list_current/remove_all"

### CODE REVIEW

File: samples/snapshot_operations.py [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+def main():
+
+    try:
+        si = None
+        try:
+            print("Trying to connect to VCENTER SERVER . . .")
+
+            context = None
+            if inputs['ignore_ssl']:
+                context = ssl.SSLContext(ssl.PROTOCOL_TLSv1)
```
Comment (tianhao64) [inline samples/snapshot_operations.py]: pls use context = ssl._create_unverified_context()

File: samples/snapshot_operations.py [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+                context.verify_mode = ssl.CERT_NONE
+
+            si = connect.Connect(inputs['vcenter_ip'], 443,
+                                 inputs['vcenter_user'], inputs[
+                                     'vcenter_password'],
+                                 sslContext=context)
+        except IOError, e:
+            pass
+            atexit.register(Disconnect, si)
```
Comment (tianhao64) [inline samples/snapshot_operations.py]: This doens't look right to me. We only disconnect when there is an exception? Catching exception is not necessary here cause there is nothing to recover here.

File: samples/snapshot_operations.py [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+
+        elif operation == 'remove_all':
+            print("Removing all snapshots for virtual machine %s" % vm.name)
+            invoke_and_track(vm.RemoveAllSnapshots())
+
+        else:
+            print("Specify operation in "
+                  "create/remove/revert/list_all/list_current/remove_all")
+
+    except vmodl.MethodFault, e:
```
Comment (tianhao64) [inline samples/snapshot_operations.py]: Catching exception is not necessary IMO.

**Comment by Abdul-Anshad-A** [comment]
Thanks for the feedback @tianhao64 . I have made the requested revisions.

**Comment by Abdul-Anshad-A** [comment]
I've made use of "WaitForTask" from "pyVim.task". Code should be much cleaner now.

**Pull request merged**